video/out/vo_mediacodec_embed.c

```diff
@@ -16,17 +16,44 @@
  */
 
 #include <libavcodec/mediacodec.h>
```

Comment:
Should remove this one. Or is it still needed?

Reply:
Still required for `AVMediaCodecBuffer` and `av_mediacodec_release_buffer`